### ISSUE TITLE
Font overhaul

### ISSUE THREAD

**Issue by aleene** [openfoodfacts/openfoodfacts-ios#392]
**Summary:** 
All interface elements now use system fonts with fixed sizes. These should be changed to the standard Text Styles for better Accessibility.

A table how these styles are used would be useful.

**Comment by aleene** [comment]
https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically

**Comment by aleene** [comment]
https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/

**Comment by aleene** [comment]
adjustsFontForContentSizeCategory = true

**Comment by aleene** [comment]
To do:
- [ ] Nutrient levels
- [ ] Nutrition facts
- [ ] Tabs
- [ ] Settings text
- [ ] Login page
- [ ] Allergen s alert
- [ ] Web-pages
